PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Uninstall Remote Assist.md

```diff
@@ -0,0 +1,53 @@
+#### Name
+
+Windows - Uninstall Remote Assist | v1.0 JCCG
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+$legacyUninstallerPath="C:\Program Files\JumpCloud\Jumpcloud Assist App\Uninstall Jumpcloud Assist App.exe"
+$uninstallerPath="C:\Program Files\JumpCloud Remote Assist\Uninstall JumpCloud Remote Assist.exe"
```

Comment:
Would these work for both x86 and x64?

Reply:
according to our [support page](https://support.jumpcloud.com/s/article/jumpcloud-agent-compatibility-system-requirements-and-impacts1#windows), we don't support 32-bit windows, but let me know if it should still be a consideration
```
As of June 30, 2020, we don't support the JumpCloud agent on 32-bit devices.
```

Reply:
Oooh nice!